src/services_models/services_constructor.jl

```diff
@@ -1,3 +1,32 @@
+function initialize_timeseries_names(
+    ::Type{<:PSY.Reserve},
+    ::Type{T},
+) where {T <: Union{RangeReserve, RampReserve}}
+    return Dict{Type{<:TimeSeriesParameter}, String}(
+        RequirementTimeSeriesParameter => "requirement",
+    )
+end
+
+function initialize_timeseries_names(
+    ::Type{<:PSY.Service},
+    ::Type{<:AbstractServiceFormulation},
+)
+    return Dict{Type{<:TimeSeriesParameter}, String}()
+end
+
+function initialize_attributes(::Type{<:PSY.Service}, ::Type{<:AbstractServiceFormulation})
```

Comment:
and this?

Reply:
This default implementation `initialize_attributes`, again I can put the dict directly in the ServiceModel constructor but that will not allow us to dispatch on different service and formulation types when we starting using attributes